lib/pom.xml

```diff
@@ -1,22 +1,25 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-        <modelVersion>4.0.0</modelVersion>
-        <groupId>pro.taskana</groupId>
-        <artifactId>taskana-parent</artifactId>
-        <version>1.1.3-SNAPSHOT</version>
-        <packaging>pom</packaging>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>taskana-lib-parent</artifactId>
```

Comment:
The Eclipse project is still named "taskana-parent", which causes some irritations now. Is it possible to change the name of the Eclipse project as well? Would it make sense to change the folders as well to be consistent?

Reply:
This seems to be a local configuration error. I removed all modules and reimportet them. Seems to be fine
![Screenshot from 2019-05-17 18-43-27](https://user-images.githubusercontent.com/15628173/57943144-8ff2d700-78d3-11e9-9aa9-485571247c45.png)

Reply:
You are right. This is stored in .project which is not checkin. I will have to correct this manually.